channel_test.go

```diff
@@ -404,7 +404,7 @@ func TestChannel_TruncateWithOptions(t *testing.T) {
 	// Now truncate it
 	_, err = ch.Truncate(ctx,
 		TruncateWithSkipPush(),
-		TruncateWithMessage(&Message{Text: "truncated channel", User: &User{ID: user.ID}}),
+		TruncateWithMessage(&Message{Text: "truncated channel", User: &User{ID: user.ID}}, user.ID),
```

Comment:
https://github.com/GetStream/stream-chat-go/pull/212/files#r841792025=

you can see it here in action via `User` in message. Let's add other option here too

Reply:
can we use the new option and see if it's set in the response?

Reply:
✅